CONTRIBUTING.md

```diff
@@ -1,36 +1,60 @@
 # Contribution Guidelines for iNaturalistReactNative
-Welcome! We're excited that you're interested in contributing to the iNaturalist React Native mobile application. Below you'll find some guidelines that will help you contribute to our project.
+Welcome! We're excited that you're interested in contributing to the iNaturalist React Native mobile application. Below you'll find some guidelines that will help you contribute to our project, whether your on staff or a volunteer.
 
-We welcome volunteer contributions! This app is still early in its development and a lot of things are in flux, but there's usually something to work on. Please keep the following in mind:
+Please keep the following in mind:
 
-1. Work on [existing issues](https://github.com/inaturalist/iNaturalistReactNative/issues) (though if you have ideas for improvement that aren't directly related to features, let us know)
-1. Leave a comment on the issue so we can assign you and other people know not to work on it
+1. Work on [existing issues](https://github.com/inaturalist/iNaturalistReactNative/issues) (feature requests should go through the [Forum](https://forum.inaturalist.org/c/feature-requests/16))
+1. Leave a comment on the issue you want to work on so we can assign you and other people know not to work on it
```

Comment:
do we care if external developers work on issues from our current iteration vs. any issues? should we steer them towards the stuff that's highest priority (or lowest?)

Reply:
Maybe we should clarify to external devs to work on unclaimed/unassigned and existing issues.

Reply:
I will specify that we want people to work on unassigned issues. I don't personally have a preference for working on issues in our current iteration or not. In theory staff have enough time to work on all the issues assigned to the current iteration, so I might actually have a slight preference for volunteers to work on issues outside of the current iteration to avoid conflicts. Do either of you have a preference?